specification/columns/skufamily.md

```diff
@@ -0,0 +1,38 @@
+# SKU Family
+
+The SKU Family column represents the common name for the family or version of the [*SKU*](#glossary:SKU).
+
+It is oftentimes the case that within a [Service Subcategory](#servicesubcategory) a [Provider](#provider) offers multiple different families of versions of SKUs. The SKU Family column provides this information to group and aggregate cost and usage data across SKUs of the same family or version.
```

Comment:
```suggestion
Typically, a [Service Subcategory](#servicesubcategory) a [Provider](#provider) offers multiple different families of versions of SKUs. The SKU Family column provides this information to group and aggregate cost and usage data across SKUs of the same family or version.
```

Reply:
I don't think we can bind SKU to ServiceSubcategory. VM bandwidth is not tied to the VM service. It's a networking charge. I'd also like to flip this around a bit...
```suggestion
SKU Family groups multiple [SKU Sizes](#skusize) that provide similar functionality. SKU Family is commonly used to group or compare prices, cost, and usage data across SKU Sizes within the same family.
```